migrate.py

```diff
@@ -217,32 +246,36 @@ def visit_Assign(self, node):
             continue
 
         # TODO what if it's in a different namespace (different spec)? do we care?
-        new_fragment = '%s.%s.%s' % (args.namespace, fragment_collection, fragment)
-        # TODO make sure to replace only in DOCUMENTATION
-        plugin_data = plugin_data.replace(fragment, new_fragment)
+        new_fragment = f'{namespace}.{fragment_collection}.{fragment}'
+
+        # TODO figure out whether this can be improved
+        doc_val.value = doc_str_tmpl.format(
+            str_val=doc_txt.replace(fragment, new_fragment),
```

Comment:
shoudl we limit replacements to 1 instance to avoid the 'multipe things rewriten'?

Reply:
What do you mean?

Reply:
I guess you mean
```suggestion
            str_val=doc_txt.replace(fragment, new_fragment, count=1),
```
But I think that there's a few things to replace there. Anyway, it worked like this before so I'd just keep it until we hit some corner case.

Reply:
the only issue i see here is if the fragment name is reused in the documentation: `default`, which creates a problem either way, we are 'replacing too much' or 'not enough', this way it should be obvious when it is the latter, the former is probably harder to detect so lets err on 'bad fragment' at this point to discover if we actually have this issue.

Reply:
really the replace should only happen inside the `extends` pragma in `DOCUMENTATION` but I'm not sure how to code that at this point.

Reply:
Should we ignore this for now?